extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/config/GrpcCodeGenConfig.java

```diff
@@ -39,4 +42,11 @@ public interface GrpcCodeGenConfig {
      */
     @WithDefault("com.google.protobuf:protobuf-java")
     String scanForImports();
+
+    /**
+     * Controls whether Kotlin code is generated when the {@code quarkus-kotlin} extension is present (in which case the default
+     * is {@code true}).
+     */
+    @WithName("kotlin.generate")
+    Optional<Boolean> generateKotlin();
```

Comment:
How about: 

```suggestion
    Optional<Boolean> kotlinGenerate();
```

Reply:
Yeah that was the default IntelliJ suggested as well but I prefer methods to read with verb first (not that it really matters at all in this case).